dbt_meshify/dbt.py

```diff
@@ -19,6 +20,12 @@ def invoke(
         if directory:
             os.chdir(directory)
         result = self.dbt_runner.invoke(runner_args if runner_args else [])
+        if not result.success and result.exception:
+            if isinstance(result.exception, UninstalledPackagesFoundError):
+                logger.debug("Project missing packages, running dbt deps...")
+                self.dbt_runner.invoke(["deps"])
+                result = self.dbt_runner.invoke(runner_args if runner_args else [])
+
```

Comment:
I really like that we'll be covered now regardless of our command. Fantastic enhancement!